cmd/shell.go

```diff
@@ -194,6 +194,8 @@ func newScope(addr string, logger blog.Logger) metrics.Scope {
 	// These handlers are defined in runtime/pprof instead of net/http/pprof, and
 	// have to be accessed through net/http/pprof's Handler func.
 	mux.Handle("/debug/pprof/goroutine", pprof.Handler("goroutine"))
+	// Sample an average of one blocking event per second spent blocked (1e9 ns)
+	runtime.SetBlockProfileRate(1e9)
```

Comment:
Is there any concern that the blocking profile rate change could cause performance problems? I would feel more comfortable if the `1e9` value was a configuration parameter and if `runtime.SetBlockProfileRate` was only called if the parameter was non-zero. That way if there is some kind of impact from this change we can revert it without a rollback of the deploy.